src/storybook/stories/Section.stories.tsx

```diff
@@ -0,0 +1,194 @@
+  args: {withFullPadding: true, backgroundColor: 'background_2'},
+  decorators: [
+    (Story, {args}) => (
+      <ScrollView>
+        <Story
+          args={{
+            ...args,
+            children: (
+              <Section>
+                <HeaderSectionItem
+                  text="HeaderSectionItem"
+                  mode="heading"
+                  transparent={false}
+                />
+                <ButtonSectionItem
+                  label="ButtonSectionItem"
+                  onPress={() => {}}
+                />
+                <ActionSectionItem
+                  text="ActionSectionItem"
+                  onPress={() => {}}
+                />
+                <CounterSectionItem
+                  text="CounterSectionItem"
+                  count={2}
+                  addCount={() => {}}
+                  removeCount={() => {}}
+                />
+                <ExpandableSectionItem
+                  text="ExpandableSectionItem"
+                  onPress={() => {}}
+                  expandContent={<ThemeText>Content</ThemeText>}
+                />
+                <FavoriteDepartureSectionItem
+                  favorite={{
+                    id: '1',
+                    stopId: '2',
+                    lineId: '3',
+                    quayId: '4',
+                    quayName: 'FavoriteDepartureSectionItem',
+                  }}
+                />
+                <FavoriteSectionItem
+                  favorite={{
+                    id: '1',
+                    location: {
+                      id: '2',
+                      name: 'FavoriteSectionItem',
+                      layer: 'address',
+                      coordinates: {longitude: 3, latitude: 4},
+                      locality: '',
+                      category: [],
+                      resultType: 'favorite',
+                      favoriteId: '5',
+                    },
+                  }}
+                />
+                <GenericClickableSectionItem>
+                  <ThemeText>GenericClickableSectionItem</ThemeText>
+                </GenericClickableSectionItem>
+                <GenericSectionItem>
+                  <ThemeText>GenericSectionItem</ThemeText>
+                </GenericSectionItem>
+                <InternalLabeledSectionItem label="InternalLabeledSectionItem">
+                  <ThemeText>Content</ThemeText>
+                </InternalLabeledSectionItem>
+                <LinkSectionItem text="LinkSectionItem" />
+                <LocationInputSectionItem
+                  label="LocationInputSectionItem"
+                  onPress={() => {}}
+                />
+                <MessageSectionItem
+                  messageType="valid"
+                  message="MessageSectionItem"
+                />
+                <PhoneInputSectionItem
+                  label="PhoneInputSectionItem"
+                  prefix="47"
+                  onChangePrefix={() => {}}
+                />
```

Comment:
Something strange with the `PhoneInputSectionItem`. Could you take a quick look into it?
![Screenshot 2024-03-01 at 09 54 01](https://github.com/AtB-AS/mittatb-app/assets/675421/203671ec-9e45-4fdc-aed6-4b33ceb2041c)

Reply:
Removed the margin on top and bottom